### DISCUSSION TITLE
Wow KTL

### DISCUSSION THREAD

**Discussion by CSWinnall** [cortexrd/Knack-Toolkit-Library#122] [General]
Hi @cortexrd 

I'd just like to say that while making our new app, with KTL from the start, has been brilliant.  

Our CSS & JavaScript is a lot leaner & with Dev mode it is speeding up development.

Thank you for your time and effort you put into it.

Craig

**Comment by cortexrd** [comment]
Hey Craig and Amanda,

You guys really rock!  Barely knowing anything about Javascript and coding in general a few months ago, then becoming the first ones to give back to the community, supporting Knack+KTL developers.

You even leveraged all those deep-down, most complex features.  I'm, so impressed.

Finally, it seems that I haven't done all this just for myself  !

I really appreciate your appreciation  LOL  ;)

Cheers,
Norm
